lightningd/json.c

```diff
@@ -115,6 +115,15 @@ struct command_result *param_feerate(struct command *cmd, const char *name,
 		if (json_tok_streq(buffer, tok, feerate_name(i)))
 			return param_feerate_estimate(cmd, feerate, i);
 	}
+	/* We used SLOW, NORMAL, and URGENT as feerate targets previously,
+	 * and many commands rely on this syntax now.
+	 * It's also really more natural for an user interface. */
+	if (json_tok_streq(buffer, tok, "slow"))
+		return param_feerate_estimate(cmd, feerate, FEERATE_MIN);
```

Comment:
We don't double it, but ECONOMICAL/100 is that far in the future and beyond that it doesn't make a big difference : we'll likely hit floor anyway.